dev-ml/ocaml-dns/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <herd>ml</herd>
+    <herd>proxy-maintainers</herd>
+    <maintainer>
+        <email>tomboy64@sina.cn</email>
+    </maintainer>
+	<longdescription lang="en">
```

Comment:
Mah bare eyes see that there is different indent near <maintainer> and <use>. You should use spaces or tabs consistently. And drop empty `<longdescription>`.